.circleci/config.yml

```diff
@@ -89,7 +89,7 @@ aliases:
       export PERL5OPT="$COVEROPT,-db,cover_db_${CIRCLE_JOB}"
       echo PERL5OPT="$PERL5OPT"
       make test-$CIRCLE_JOB
-    no_output_timeout: 30m
+    no_output_timeout: 50m
```

Comment:
In which case do 50 minutes with no output apply? No individual test will take that long, even with output it will time out

Reply:
I am sorry. I was missing some links in the PR for ference. Please see https://progress.opensuse.org/issues/71554#note-5 for example

Reply:
Right, that's actually the "no output" timeout. We actually shouldn't run into this.

Reply:
I don't think anyone wants to argue - I just don't follow what your aim is here. Are you suggesting tests can legitimately get stuck with no output for 30 or even 50 minutes?

Reply:
Well, you introduced the change to not show any useful output in the log window so apparently that is the case.

I merely want to achieve one little thing with this PR: circle_ci_no_output_timeout > sum(RETRIES * timeout per retry) > timeout within t/full-stack.t